packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -291,33 +300,46 @@ function Pending(props) {
         </PendingButtonContainerComponent>
       </Flex>
 
-      <View flexBasis={'100%'} paddingTop={16}>Controlled:</View>
+      <Flex wrap="wrap" alignItems={'center'}>
+        <Text>Aria-label "Button label" on button</Text>
+        <PendingButtonContainerComponent {...props}>
+          <PendingButtonComponent
+            {...props}
+            aria-label="Button label">
+            <Bell />
+          </PendingButtonComponent>
+        </PendingButtonContainerComponent>
 
-      <Flex wrap="wrap">
+        <Text>Aria-label "icon label" on icon</Text>
         <PendingButtonContainerComponent {...props}>
-          <TooltipTrigger offset={2}>
-            <Button {...props} aria-label="Notifications" onPress={() => {window.alert('use storybook control to change this button isPending prop');}}>
-              <Bell />
-            </Button>
-            <Tooltip>Notifications</Tooltip>
-          </TooltipTrigger>
+          <PendingButtonComponent {...props}>
+            <Bell aria-label="icon label" />
+          </PendingButtonComponent>
         </PendingButtonContainerComponent>
 
+        <Text>No aria-labels--bad implementation</Text>
         <PendingButtonContainerComponent {...props}>
-          <Button {...props} aria-label="No tooltip" onPress={() => {window.alert('use storybook control to change this button isPending prop');}}>
+          <PendingButtonComponent {...props}>
             <Bell />
-          </Button>
+          </PendingButtonComponent>
         </PendingButtonContainerComponent>
 
+        <Text>Tooltip and aria-label "Notifications" on button</Text>
         <PendingButtonContainerComponent {...props}>
-          <Button {...props} onPress={() => {window.alert('use storybook control to change this button isPending prop');}}>
-            <Bell />
-          </Button>
+          <TooltipTrigger offset={2}>
+            <Button {...props} isPending={tooltipPending} onPress={handlePress} aria-label="Notifications">
```

Comment:
Why does this instance have a custom onPress?

Reply:
all of them do, this one just isn't wrapped up like the others. I assume it was added after the others and needed more props and it was easier to just do inline.